Client/Frontend/Library/HistoryPanel/HistoryPanel.swift

```diff
@@ -183,6 +183,7 @@ class HistoryPanel: UIViewController,
 
         // Update theme of already existing view
         bottomStackView.applyTheme(theme: themeManager.currentTheme)
+        viewModel.shouldResetHistory = true
```

Comment:
Wondering if this should be behind the `CoordinatorFlagManager.isLibraryCoordinatorEnabled` flag if we keep this (see other comments)